spec/mongoid/query_cache_spec.rb

```diff
@@ -10,6 +11,20 @@
     Mongoid::QueryCache.cache { spec.run }
   end
 
+  before(:all) do
+    # It is likely that there are other session leaks in the driver
+    # and/or Mongoid that are unrelated to the query cache. Clear the
+    # SessionRegistry at the start of these tests in order to detect leaks that
+    # occur only within the scope of these tests.
+    #
+    # Other session leaks will be detected and addressed as part of RUBY-2391.
+    SessionRegistry.instance.clear_registry
+  end
+
+  before do
```

Comment:
Shouldn't this be `after` ?

Reply:
Also a good catch! I think I was having trouble running it in an after block at some point, so I switched it and then never switched it back. It works now though.